js/main.js

```diff
@@ -1,5 +1,74 @@
+import {Player} from './player.js';
+import {Game} from './game.js';
 let play = document.querySelector('#start-game');
+let cells  = document.querySelectorAll('.cell');
+let label = document.querySelector('#player-name');
+let replay = document.querySelector('#replay');
+let formContainer = document.querySelector('#hidden');
+let winMessage = document.querySelector('#winner');
```

Comment:
You are polluting the global object.
Encapsulate these statements inside the functions or factories that these variables are being used.